src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/DropIndexAttemptBuilder.java

```diff
@@ -0,0 +1,31 @@
+package io.stargate.sgv2.jsonapi.service.operation.tables;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.KeyspaceSchemaObject;
+import io.stargate.sgv2.jsonapi.util.CqlIdentifierUtil;
+
+/** Builds a {@link DropIndexAttempt}. */
+public class DropIndexAttemptBuilder {
+  private int position;
+  private final KeyspaceSchemaObject schemaObject;
+  private CqlIdentifier name;
+  private boolean ifExists;
+
+  public DropIndexAttemptBuilder(KeyspaceSchemaObject schemaObject) {
+    this.schemaObject = schemaObject;
+  }
+
+  public DropIndexAttemptBuilder withName(String name) {
```

Comment:
If "name" is not optional (unlike "ifExists"), should be passed via constructor.